.github/actions/process_recipe/action.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'Process Recipe'
```

Comment:
I'm a bit confused why the action itself is being put into staged-recipes. Shouldn't a github action live in its own standalone repo? Pls explain.

Reply:
See my below comment.